src/styles/blocks/stopwatch.scss

```diff
@@ -0,0 +1,70 @@
+$stopwatch-size: 80vmin;
+$stopwatch-center: $stopwatch-size / 2;
+$center-size: 5vmin;
+$center-center: $center-size / 2;
+$minutes-hand-width: 3vmin;
+$minutes-hand-heigh: 20vmin;
```

Comment:
There is a typo in the variable name `$minutes-hand-heigh`. It should be `$minutes-hand-height` to maintain consistency and clarity.